plugins.d/Mail_Relaying/mail_relay.py

```diff
@@ -23,34 +25,44 @@
 
 
 def testsettings(host, port, login, password):
-    host = host.encode('utf-8')
+
+    encoding = sys.stdin.encoding
+
+    # login username and password must be string
+    def bytes2string(string):
+        if type(string) is bytes:
```

Comment:
Generally ``isinstance(string, bytes)`` would be prefered to ``type(string) == bytes`` or ``type(string) is bytes`` as it handles inheritance, although it **probably** doesn't really matter here

Reply:
Hmm. Ok thanks... I'll leave it be for now anyway.